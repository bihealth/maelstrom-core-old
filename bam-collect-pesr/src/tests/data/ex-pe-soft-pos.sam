../../../../bam-scan/src/tests/data/ex-pe-soft-pos.sam