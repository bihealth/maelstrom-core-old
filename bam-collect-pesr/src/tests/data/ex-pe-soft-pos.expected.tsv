#contig	start	end	signal
one	299	379	{"SplitRead":{"read_name":"pair_expected","is_first":false,"contig":"one","start":299,"end":379,"clipped_sides":"Right"}}
